Libs/MRML/Core/vtkMRMLUnitNode.h

```diff
@@ -99,14 +111,30 @@ class VTK_MRML_EXPORT vtkMRMLUnitNode : public vtkMRMLNode
   vtkSetStringMacro(Prefix);
 
   ///
-  /// Set/Get the unit suffix. For example, the suffix for the unity
-  /// meter would be "m".
+  /// Set/Get the unit suffix. For example, the suffix
+  /// for the unity metre would be "m".
   /// Default is "".
   /// \sa SetSuffix(), GetSuffix()
   vtkGetStringMacro(Suffix);
   vtkSetStringMacro(Suffix);
 
   ///
+  /// Set/Get the unit secondsuffix. For example, the secondsuffix
+  /// for the unity degree would be "\x27".
+  /// Default is "".
+  /// \sa SetSecondSuffix(), GetSecondSuffix()
+  vtkGetStringMacro(SecondSuffix);
+  vtkSetStringMacro(SecondSuffix);
+
+  ///
+  /// Set/Get the unit thirdsuffix. For example, the thirdsuffix
+  /// for the unity degree would be "\x22".
```

Comment:
X27 and x22 are primes, ',  (60' =  1°) and arcseconds, ", (60" = 1'). For us the unit degree is indeed the angle of view on the sky. I can change the  description, if you think that is not explanatory.